platform/stm32f7xx/qspi.c

```diff
@@ -375,7 +377,16 @@ static ssize_t spiflash_bdev_erase(struct bdev* device, off_t offset,
 
 static int spiflash_ioctl(struct bdev* device, int request, void* argp)
 {
-    return ERR_NOT_IMPLEMENTED;
+    int ret = ERR_NOT_SUPPORTED;
+
+    switch (request) {
+        case BIO_IOCTL_GET_MEM_MAP:
```

Comment:
Doesn't support writing while in linear mode.

Will add that in the next commit.